src/box/lua/net_box.lua

```diff
@@ -7,6 +7,8 @@ local urilib   = require('uri')
 local internal = require('net.box.lib')
```

Comment:
I agree this looks weird, but we do this in quite a few tests:

```
$ grep -RI net:connect test | wc -l
45
```

Since it's trivial to support, I think we'd better leave it as is.